event/request.go

```diff
@@ -0,0 +1,96 @@
+package event
```

Comment:
can you move the content of this file to event.go? those file are pretty short and the are basically about the same thing.